### ISSUE TITLE
Form and Interactive Instrument need basic templates

### ISSUE THREAD

**Issue by gdevenyi** [DouglasNeuroInformatics/OpenDataCapture#661]
The form playground needs to have a basic "empty" template for both these types which contains the minimum acceptable fields for the instruments.

**Comment by joshunrau** [comment]
Do you want this in English only or bilingual?

**Comment by gdevenyi** [comment]
Both?

There's slightly different minimum requirements right?

**Comment by joshunrau** [comment]
> Both?
> 
> There's slightly different minimum requirements right?

Not really, it is just the structure of the required fields that is different. I put this online so you can see for yourself.

**Comment by gdevenyi** [comment]
This is great. Thanks.
